app/components/specific/map/RaceMap.js

```diff
@@ -103,11 +124,24 @@ export default class RaceMap extends Component {
               athlete => athlete.id === parseInt(id, 10)
             );
 
+            let description = "";
+            const index = race.path.coordinates.findIndex(
```

Comment:
this seems wrong .
i think you want to get the path done?

Reply:
nearestPoint is the closest point of event map path to athlete current position.  
So this point is for sure one item of race map path.